cloud_utils/scheduler/kubernetes_connector.py

```diff
@@ -0,0 +1,181 @@
+import base64
+import logging
+import os
+from pathlib import Path
+from typing import Dict, List, Text
+
+import gamla
+import toolz
+from kubernetes import client, config
+from kubernetes.client import rest
+
+repo_name = os.getenv("CI_REGISTRY")
```

Comment:
this looks fine. but let's pass it as an argument to our tool

e.g. automation deploy $CI_REGISTRY

Reply:
Sure, I'm also adding path to scheduling file as optional parameter